pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -92,10 +92,21 @@ func (h *DHCPv6Handler) ServeDHCPv6(conn net.PacketConn, peer net.Addr, m dhcpv6
 
 	// TODO if we extend the server to support bridge binding, we need to filter out non-vm requests
 
+	response, err := h.buildResponse(m)
```

Comment:
I didn't leak to the production code. Those changes were done so it will be easier to unit test the dhcpv6 server. Please refer to my answer on another [commit](https://github.com/kubevirt/kubevirt/pull/4654/commits/1a6c46a404cd2b7920dd406a7fcc16e30643065e#r545357572).